test/test_parse.py

```diff
@@ -71,20 +71,105 @@ def test_articles(self):
         self.assertEqual(extract_number("this is the first test",
                                         ordinals=True), 1)
-        self.assertEqual(extract_number("this is 2 test"), 2)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("this is the first test",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/time unit
         self.assertEqual(extract_number("this is second test",
                                         ordinals=True), 2)
-        self.assertEqual(extract_number("this is the third test"), 1.0 / 3.0)
+        self.assertEqual(extract_number("this is second test",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=True), 2)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=False), False)
+        self.assertEqual(extract_number("remind me in a second",
+                                        ordinals=None), False)
+
+        # test ambiguous ordinal/fractional
         self.assertEqual(extract_number("this is the third test",
                                         ordinals=True), 3.0)
+        self.assertEqual(extract_number("this is the third test"),
+                         1.0 / 3.0)
+        self.assertEqual(extract_number("this is the third test",
+                                        ordinals=None), False)
+
+        self.assertEqual(extract_number("one third of a cup"), 1.0 / 3.0)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=True), 3)
+        self.assertEqual(extract_number("one third of a cup",
+                                        ordinals=None), 1)
+
+        # test plurals
+        # NOTE plurals are never considered ordinals, but also not
+        # considered explicit fractions
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=True), 2)
+        self.assertEqual(extract_number("2 fifth",
+                                        ordinals=True), 5)
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=False), 2/5)
+        self.assertEqual(extract_number("2 fifths",
+                                        ordinals=None), 2)
+
+        self.assertEqual(extract_number("Twenty two and Three Fifths"), 22.6)
+
+        # TODO fix me - see issue #152
+        # these currently return 22 instead of 3
+        #self.assertEqual(extract_number("Twenty two and Three Fifths",
+        #                                ordinals=True), 3)
+
+        # test multiple ambiguous
+        self.assertEqual(extract_number("sixth third"),
+                         1 / 6 / 3)
```

Comment:
I don't think this one makes sense. 
"Sixth third" would probably refer to "the sixth instance of a third" which I guess means there's `2` all up.

Reply:
Agreed. I dunno what should be returned here, but I don't think it should be cumulative.

Reply:
this one has been around for ages, i just changed it to a different place so all related tests are grouped together

i dont think it makes a lot of sense either, but i'm declaring it out of scope since it isnt touched by this PR

Reply:
added a code comment for this one